### ISSUE TITLE
Question RE LongituRF Models from your 2021 Blog Post

### ISSUE THREAD

**Issue by ghost** [Chhr1s/Distill_Website#5]
Good morning Christopher,

I hope you are well. I'm sure this isn't the most appropriate place to ask this question, but I was unable to find your email on your website! I have been using LongituRF to fit mixed-effects random forest models for a paper, and used your website's guide (extremely helpful.

One issue I have experience, though, is predicting on a test set with the predict.longituRf() function using a training set. Specifically, if I use a training set to fit a MERF using 75% of the original dataset, when I try to predict on my test set, I get the following error message: 

Error in Ypred[w] <- f[w] + Z[w, , drop = FALSE] %*% object$random_effects[k, :
replacement has length zero

What's interesting is when I use a 50/50 train/test split, I don't get the error message, and the predict function works just fine. I have a hunch that I could use your modified predict function to fix this issue, but have hit a bit of a wall!

Any insight you could possibly provide would be greatly appreciated.

Thank you so much, and again, sorry for hacking into your GitHub to send this haha

- Chris

**Comment by Chhr1s** [comment]
Hello, I can’t help based on the info provided. I looked at your GitHub and saw you gave more details on the package’s reported issue section. My best guess is that you have error upstream to this based on your sample size. However, really no certainty with the available info. Do you have an email address I can reach out to?

**Comment by christopherDennison** [comment]
Afternoon Christopher,

Thanks so much for your time! Sure thing. My email is: ***@***.***

Happy to provide any details, code or data that might help.


  *
Chris🦉

Christopher Dennison
MSc. Student, Biology
Carleton University
1125 Colonel By Drive
Ottawa, Ontario
email: ***@***.***
phone: (613)-581-9361
________________________________

**Comment by christopherDennison** [comment]
christopherdennison@cmail.carleton.ca

**Comment by Chhr1s** [comment]
Hi, I got your email and will reach out. You might want to delete the comment here so it’s not public. You also might not care! I’ll email rn
